src/jdk.incubator.code/share/classes/jdk/incubator/code/internal/ReflectMethods.java

```diff
@@ -154,6 +155,8 @@ protected ReflectMethods(Context context) {
         lineDebugInfo =
                 options.isUnset(G_CUSTOM) ||
                         options.isSet(G_CUSTOM, "lines");
+        String cmso = options.get("codeModelStorageOption");
+        codeModelStorageOption = CodeModelStorageOption.valueOf(cmso != null ? cmso : CodeModelStorageOption.TEXT.name());
```

Comment:
If you want you can define the parsing code here as a static method in the CodeModelStorageOption enum, for better encapsulation.